src/services/sync/index.js

```diff
@@ -54,7 +55,10 @@ module.exports = function setup() {
 
 	app.use('/sync/api', staticContent(path.join(__dirname, '/docs/openapi.yaml')));
 	app.use('/sync', new SyncService());
-	app.configure(consumer);
+
+	if (Configuration.get('FEATURE_RABBITMQ_ENABLED')) {
```

Comment:
RabbitMQ can not disabled if ldap is set and used it. We must find a way to bind this in the config structure.